src/sql/util.py

```diff
@@ -10,14 +10,18 @@
 import ast
 from os.path import isfile
 import re
+import sys
 
 from jinja2 import Template
 
 
 try:
     import toml
 except ModuleNotFoundError:
-    toml = None
+    if sys.version_info >= (3, 11):
+        import tomllib as toml
```

Comment:
does tomlib have the same api as the toml package?

also, I think more changes are needed because the function that loads toml, still checks that the package is installed, and has a try-catch statement that checks for an error:

```

@requires(["toml"])
def load_toml(file_path):
    """
    Returns toml file content in a dictionary format
    and raises error if it fails to load the toml file
    """
    try:
        with open(file_path, "r") as file:
            content = file.read()
            return toml.loads(content)
    except toml.TomlDecodeError as e:
        raise parse_toml_error(e, file_path)
```

Reply:
Regarding the API, for the `loads` function, then both `toml` and `tomllib` have the same API. However, the exception is different, `TomlDecodeError` vs `TOMLDecodeError`. 

Thanks for pointing this out, let me take another crack at this.